src/main/java/com/example/enterpriseapplication/AddRecipeController.java

```diff
@@ -6,30 +6,31 @@
 import org.springframework.web.bind.annotation.RequestParam;
 
 import com.example.enterpriseapplication.dto.Recipe;
-import com.example.enterpriseapplication.service.RecipeService;
+import com.example.enterpriseapplication.service.IRecipeService;
```

Comment:
Corrected the import to reference the right service.

https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3